### PR TITLE
Create front page "feature cards"

### LINKED ISSUE #39: Create a "Featured" section at the top of PlasmaPy's landing page
Create a section/banner at the top of https://www.plasmapy.org to highlight news/events we want to broadcast.  This could be a revolving banner or a set of featured tiles.

I can envision featured items being...

1. Latest version release
1. Project support/funding (i.e. CSSI)
1. Our weekly meetings
1. Upcoming workshops
1. etc.

### PR DESCRIPTION
**Pull request by rocco8773**
This creates a set of 3 "feature cards" at the top of the PlasmaPy landing page.  These cards can be used to highlight things like new releases, upcoming meetings, funding sources, etc.

PR #56 should be merged first
Closes #39

### CODE REVIEW

File: web/files/assets/css/custom.css [hunk 1]
```diff
@@ -38,26 +38,121 @@
...
+    position: relative;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    padding: 16px;
+    background-color: #f1f1f1;
+    border-style: solid;
+    border-color: #f1f1f1;
+    border-width: 2px;
+    height: 200px;
+
+    background-image: linear-gradient(rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0.5)), url("/images/favicon-128.thumbnail.png");
```
Comment (StanczakDominik) [inline web/files/assets/css/custom.css:133]: Ah, I was wondering why it looked a bit low res - maybe use https://github.com/PlasmaPy/PlasmaPy-logo/blob/master/exports/graphic-circular.png instead?

Comment (rocco8773) [reply web/files/assets/css/custom.css:133]: yes, that is the reason

File: web/files/assets/css/custom.css [hunk 2]
```diff
@@ -120,38 +102,48 @@ table tr th :last-child, table tr td :last-child {
...
 }
 
+/* Graphics of the "feature card" to be placed in the .feature-column
+ *   Intended to be used with <span> so it can be nested inside a
+ *   hyperlink <a>
+ *
+ *   By default the PlasmaPy graphic will be placed as a background
+ *   image, but this can be changed by specifying a new value for the
+ *   background-image property.
+ */
```
Comment (StanczakDominik) [inline web/files/assets/css/custom.css:112]: As someone to whom css is magic, thank you, o great wizard!

File: web/files/assets/css/custom.css [hunk 3]
```diff
@@ -120,38 +102,48 @@ table tr th :last-child, table tr td :last-child {
...
+
     border-style: solid;
     border-color: #f1f1f1;
     border-width: 2px;
-    height: 200px;
+    border-radius: 10px;
 
-    background-image: linear-gradient(rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0.5)), url("/images/favicon-128.thumbnail.png");
+    background-color: #f1f1f1;
+    background-image: linear-gradient(rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0.5)), url("/images/plasmapy-graphic-circular.png");
```
Comment (StanczakDominik) [inline web/files/assets/css/custom.css:126]: You forgot to `git add` the .png!

File: web/files/assets/css/custom.css (hunk 2 above)
Comment (rocco8773) [reply web/files/assets/css/custom.css:113]: Yeah, it's always been magic to me too.  This site helped me so much...

https://www.w3schools.com/howto/default.asp

File: web/files/assets/css/custom.css (hunk 3 above)
Comment (rocco8773) [reply web/files/assets/css/custom.css:127]: *facepalm*

**Comment by StanczakDominik** [comment]
Looks great! :smile: Thanks, @rocco8773!

**Pull request merged**